pkg/util/runtimeconfig/manager.go

```diff
@@ -44,20 +45,26 @@ type Manager struct {
 	config    interface{}
 
 	configLoadSuccess prometheus.Gauge
+	configInfo        *prometheus.GaugeVec
 }
 
 // NewRuntimeConfigManager creates an instance of Manager and starts reload config loop based on config
 func NewRuntimeConfigManager(cfg ManagerConfig, registerer prometheus.Registerer) (*Manager, error) {
 	mgr := Manager{
 		cfg: cfg,
 		configLoadSuccess: prometheus.NewGauge(prometheus.GaugeOpts{
-			Name: "cortex_overrides_last_reload_successful",
-			Help: "Whether the last config reload attempt was successful.",
+			Name: "cortex_runtime_config_last_reload_successful",
+			Help: "Whether the last runtime-config reload attempt was successful.",
```

Comment:
I have changed that because i found this mismatch between CLI argument and metric name quite confusing. 

Not too sure if this is subject to a deprecation process or if we just should use the `cortex_overrides` for both

Reply:
I think it's OK to change (if you mention it in CHANGELOG.md and update existing alerts too in https://github.com/grafana/cortex-jsonnet repo) but I'm not sure if metric names are actually part of our V1 guarantees. It's not mentioned in the docs/configuration/v1-guarantees.md doc. @gouthamve what do you say?

Reply:
Checked with Goutham, we don't cover metrics in guarantees, so it's ok to rename it. (with changelog and alerts update)

Reply:
Yep, following this lets make the changes in the [cortex-mixin](https://github.com/grafana/cortex-jsonnet) as well.

Reply:
@simonswine As Annanay correctly mentioned, please don't forget to open a PR to update the mixin once this PR will be merged, thanks! 🙏 🙏 🙏